src/components/Docs/SnippetViewer/WriteAuthzModelViewer.tsx

```diff
@@ -22,7 +22,7 @@ function writeAuthZModelViewerCurl(authorizationModel: AuthorizationModel): stri
 
 function writeAuthZModelViewerJS(authorizationModel: AuthorizationModel): string {
   return `
-const { authorization_model_id: id } = await fgaClient.writeAuthorizationModel(${JSON.stringify(
+const { authorizationModelId: id } = await fgaClient.writeAuthorizationModel(${JSON.stringify(
```

Comment:
```suggestion
const { authorization_model_id: id } = await fgaClient.writeAuthorizationModel(${JSON.stringify(
```

This is a response, not a request

Reply:
Yeah but shouldn't the JS variable be called authorizationModelId if we want to comply to JS naming conventions?

Reply:
It should, but currently we do that only for client inputs, the outputs still use the API models.

This is documented here: https://github.com/openfga/js-sdk?tab=readme-ov-file#calling-the-api

Reply:
Right, I get it now